### PR TITLE
changed response.message() to call toString() instead

### LINKED ISSUE #8042: Improve 404 error messages (WorldClient, FeedClient...)
My branch was outdated and the WorldClient error message didn't show that I was getting a 404.

```
2019-02-28 18:44:14 [Thread-3] DEBUG n.r.http.api.worlds.WorldClient - Error looking up worlds: 
```

A quick fix is changing the `response.message()` in `WorldClient.java:60` to `response.toString()` so the full error is shown:
```
2019-02-28 18:44:14 [Thread-3] DEBUG n.r.http.api.worlds.WorldClient - Error looking up worlds: Response{protocol=h2, code=404, message=, url=https://api.runelite.net/runelite-1.5.14-SNAPSHOT/worlds.js}
```

However, other classes also throw empty error messages, like FeedClient. A different approach might be to populate the message when it's empty and a specific response code is found.

### PR DESCRIPTION
**Pull request by ndngo**
Closes #8042

### CODE REVIEW

File: http-api/src/main/java/net/runelite/http/api/feed/FeedClient.java [hunk 1]
```diff
@@ -55,7 +55,7 @@ public FeedResult lookupFeed() throws IOException
 		{
 			if (!response.isSuccessful())
 			{
-				logger.debug("Error looking up feed: {}", response.message());
+				logger.debug("Error looking up feed: {}", response.toString());
```
Comment (chivesrs) [inline http-api/src/main/java/net/runelite/http/api/feed/FeedClient.java]: Doesn't the logger call toString() natively on the object passed in? It seems redundant to have a toString() here if that's the case.

Ref: https://github.com/runelite/runelite/issues/8042#issuecomment-468382802

Comment (deathbeam) [reply http-api/src/main/java/net/runelite/http/api/feed/FeedClient.java]: It does

**Comment by ndngo** [comment]
Ah, I see. That makes sense.

File: http-service/src/main/java/net/runelite/http/service/feed/twitter/TwitterService.java [hunk 2]
```diff
@@ -146,7 +146,7 @@ private void updateToken() throws IOException
 		{
 			if (!response.isSuccessful())
 			{
-				throw new IOException("Error authing to Twitter: " + response.message());
+				throw new IOException("Error authing to Twitter: " + response.toString());
```
Comment (deathbeam) [inline http-service/src/main/java/net/runelite/http/service/feed/twitter/TwitterService.java]: This still has toString call

**Pull request merged**